### PR TITLE
Add and adjust integration test for lookup

### PR DESCRIPTION
**Pull request by TobiasNx**
Related to:
metafix/src/test/java/org/metafacture/metafix/MetafixLookupTest.java

I added an integration test for test @blackwinter has added in context of #102

### CODE REVIEW

File: metafix/src/test/resources/org/metafacture/metafix/integration/lookup/fromJson/toJson/lookupInArrayOfStringsWithNotMatchingValues/expected.json [hunk 1]
```diff
@@ -2,8 +2,14 @@
   "animals" : [ "mammal", "bird", "fish" ]
 }
 {
-  "animals" : [ "human" ]
+  "animals" : [ "human", "bird", "fish" ] ]
```
Comment (blackwinter) [inline metafix/src/test/resources/org/metafacture/metafix/integration/lookup/fromJson/toJson/lookupInArrayOfStringsWithNotMatchingValues/expected.json:5]: Syntax error.

**Review by blackwinter — APPROVED** [review]
Shouldn't the names of the two tests be swapped? `lookupInArrayOfStringsDeleteNotMatchingValues` has `delete` in its name, but not in the Fix. `lookupInArrayOfStringsWithNotMatchingValues` has `delete` in the Fix, but not in its name.

**Comment by TobiasNx** [comment]
I changed the folder names, @blackwinter. Can I merge now?

**Comment by blackwinter** [comment]
Yes, I've already approved the pull request.

**Pull request merged**